nncf/torch/automl/environment/quantization_env.py

```diff
@@ -281,8 +287,21 @@ def _create_quantizer_table(self) -> pd.DataFrame:
         assert len(set(self.qconfig_space_map.keys()) - set(adjq_gid_map.keys())) == 0, \
             "both qconfig_space_map and adjq_gid_map must have exact keys."
 
+        # AutoQ requires quantizers in execution order
+        quantizers_in_exec_order = OrderedDict()
+        hooklist = []
+        for qid, qmod in self.qctrl.all_quantizations.items():
+            hooklist.append(
+                qmod.register_forward_hook(
+                    get_hook((qid, qmod), quantizers_in_exec_order)
+                )
+            )
+        self.qmodel.do_dummy_forward(force_eval=True)
+        for h in hooklist:
+            h.remove()
+
         d = OrderedDict()
-        for qid in self.qctrl.all_quantizations:
+        for qid, qmod in quantizers_in_exec_order.items():
```

Comment:
do you really need qmod? 
just wonder why do you need OrderedDict, instead of simple List of qid?

Reply:
we dont. I've patched it :)